java/src/org/certificatetransparency/ctlog/MerkleAuditProof.java

```diff
@@ -0,0 +1,18 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class MerkleAuditProof {
+  public Ct.Version version;
+  public long treeSize;
+  public long leafIndex;
+  public java.util.List<byte[]> pathNode;
+
+  public MerkleAuditProof(Ct.Version version_, long treeSize_, long leafIndex_) {
+    version = version_;
+    treeSize = treeSize_;
+    leafIndex = leafIndex_;
+    pathNode = new java.util.ArrayList<byte[]>();
```

Comment:
Is that a reasonable implementation of `List<>` to use? I just looked around and poked in the dark until it compiled and the tests passed, really... :wink: